Public demo of field-colour-hsv-sliders does not launch

**Category**

 - Plugins

**Component**

`field-colour-hsv-sliders`

**Describe the bug**

The public demo has `Uncaught Error: Cannot find module '@blockly/field-colour'` and does not launch the demo.

**To Reproduce**

Open https://google.github.io/blockly-samples/plugins/field-colour-hsv-sliders/test/index.html

**Expected behavior**

Working demo.

**Screenshots**

![image](https://github.com/google/blockly-samples/assets/12326241/11f430d9-4918-40c1-a441-00ec64b01b63)

**Additional context**

Possibly related to imports as in https://github.com/google/blockly-samples/issues/1892

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "clean:node": "lerna clean --yes",
     "deploy:prepare": "npm run deploy:prepare:plugins && npm run deploy:prepare:examples && gulp predeploy",
     "deploy:prepare:examples": "cd examples && npm run predeploy",
-    "deploy:prepare:plugins": "npm run clean && lerna run predeploy",
+    "deploy:prepare:plugins": "npm run clean && lerna run build && lerna run predeploy",
     "deploy": "npm run deploy:prepare && gulp deploy",
     "deploy:upstream": "npm run deploy:prepare && gulp deployUpstream",
     "license": "gulp checkLicenses",

